BUILD

```diff
@@ -26,3 +26,12 @@ filegroup(
         "//:__subpackages__",
     ],
 )
+
+config_setting(
+    name = "supports_visionos",
+    values = {"define": "supports_visionos=1"},
```

Comment:
Any reason why not go with a `bool_flag` here?

Reply:
if you do that it looks like you have to also still have a config_setting to use it in select statements

Reply:
Generally we shouldn't use `--define`, as it's essentially deprecated in favor of build settings.

Reply:
note that this is internal only. done